internal/cluster/cmd/clusterd/wallet.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"math"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/coreutils/chain"
+	"go.sia.tech/coreutils/testutil"
+	"go.sia.tech/coreutils/wallet"
+)
+
+type swallet struct {
```

Comment:
could avoid `swallet` which is a little odd imo by aliasing coreutils wallet as `cwallet` here